server/routes/events.ts

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+import asyncHandler from 'express-async-handler';
+
+import eventsController from 'server/controllers/eventsController';
+
+const router = express.Router();
+
+router.get(
+  '/api/chapters/:chapter_id/events',
+  asyncHandler(eventsController.index),
+);
+router.post(
+  '/api/chapters/:chapter_id/events',
```

Comment:
Why the break from JavaScript convention with the snake case?

Reply:
My soul still belongs to rails :light_rail: :gem: :rofl:, will fix

Reply:
Although we're using snake_case pretty much everywhere in the api, so maybe we should use it here to keep it consistent?

Reply:
I disagree. I think we should go camelCase here and create a tech debt ticket to convert the snake_case variables to camelCase.

However, am open to hear what others think.